backend/python/app/services/implementations/story_service.py

```diff
@@ -145,6 +148,8 @@ def get_story_translations(
                 filters.append(Story.level == level)
             if story_title is not None:
                 filters.append(Story.title.like(f"%{story_title}%"))
+            if role_filter is not None:
+                filters.append(role_filter)
```

Comment:
out of curiosity, why the change in order of applying filters?

Reply:
LOL so I had initially removed the role_filter filter while debugging a GQL error but I realized that that wasn't what was causing it and added it back in. I just followed the order in which the parameters were listed above.